### PR TITLE
chore(repo): add design defect issue template

### PR DESCRIPTION
**Pull request by tay1orjones**
Adds a new issue template geared towards design defects. 

@laurenmrice pointed out how the existing bug report can be difficult for non-devs to fill out since it requires the codesandbox link. This new template replaces that field with a new field for screenshots.

Let me know if there's other things we should add or remove to improve this.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/DESIGN_DEFECT.yaml [hunk 1]
```diff
@@ -0,0 +1,93 @@
+name: Design defect 🎨
```
Comment (tay1orjones) [inline .github/ISSUE_TEMPLATE/DESIGN_DEFECT.yaml:1]: Using :art: or :rainbow:  or 🖼️  to denote "design" always feels sort of cheap to me. If anyone has an emoji they prefer, let me know!

**Review by aagonzales — APPROVED** [review]
Looks good! The 🎨 icon is fine

**Review by laurenmrice — APPROVED** [review]
Nice! Thank you for doing this so quickly! 🎉

**Review by alisonjoseph — APPROVED** [review]
🎉

**Pull request merged**